package.json

```diff
@@ -42,14 +42,14 @@
   },
   "license": "LGPL-3.0",
   "scripts": {
+    "prebuild": "npm run lint",
     "build": "npm run build:main && npm run build:standalone",
     "build:main": "browserify -e ./src/main.js -t deamdify | derequire - > index.js",
     "build:standalone": "browserify -d -e ./src/main.js -t deamdify -s peaks | exorcist peaks.js.map | derequire - > peaks.js",
     "doc": "jsdoc --private --destination docs --recurse src",
     "lint": "eslint src/**/*.js test/**/*.js",
     "pretest": "npm run build",
     "test": "./node_modules/karma/bin/karma start",
-    "posttest": "npm run lint",
```

Comment:
@chrisn it is actually better to me to consider the linting as being a pre or post test task. Build can in fact happen without a harm if eslint rules do not comply.

Having it in `prebuild` is an implicit assumption the `build` step happens at `test` time – which can change if the browserify karma plugin is enabled for example (no need to build anymore).

What do you think?